.circleci/config.yml

```diff
@@ -19,15 +19,6 @@ commands:
           name: "Starting local registry"
           command: docker run -d -p 5000:5000 --restart=always --name registry registry:2
       - install-singularity-requirements
-      - when:
-          condition:
-            or:
-            - equal: [ "", "<< parameters.variant >>" ]
-            - equal: [ "lite", "<< parameters.variant >>" ]
-          steps:
-            - run: 
-                name: "Cloning Singularity 2.5.2"
-                command: git clone -b 2.5.2 https://github.com/sylabs/singularity
```

Comment:
Double-checking: are these intentional circleCI config changes or pulled from a commit thread somewhere? Want to make sure we don't clobber recent changes.

Reply:
No biggie, just needed a merge conflict resolution, done in https://github.com/FCP-INDI/C-PAC/pull/1492/commits/e1ac418069f3bf874b31218827d8f9cb9f0b094f

Reply:
At the moment, ABCD-HCP variant isn't included in the combined coverage because [it takes too long to attach 3 Docker images and 3 Singularity images in the fanned-in workspace](https://app.circleci.com/pipelines/github/shnizzedy/C-PAC/403/workflows/a7c111da-331c-49dc-b1ad-9517bca124d5). I'm [trying out caching the coverage files instead of using workspaces there](https://app.circleci.com/pipelines/github/shnizzedy/C-PAC/404/workflows/106dba0d-8582-4a30-a617-a3a9786163e0).

Reply:
#1498